### DISCUSSION TITLE
Setting estimate as Stable

### DISCUSSION THREAD

**Discussion by daevadragon** [mclemente/healthEstimate#190] [Q&A]
Does anyone know how I could set the module to read the estimate as stable once a creature has stabilized? Is this at all supported in the current functionality? Willing to use macros but have no idea where to start writing them.

Edit: Tried using new estimate with a JS rule `actor.effects.find((effect) => effect.statuses.has("stable"))` but found that setting new estimate section  with any JS rule removes the default estimate, opening separate issue

**Comment by mclemente** [comment]
I've just answered your issue (#192) and the provided code should now work fine.

**Reply by daevadragon** [reply]
Thanks so much for addressing that so quickly. I am on Forge and so don't believe I'll be able to see the functionality until a new update pushes through there, unless there's another way I should update the module, but I really appreciate your timely response to this issue

**Reply by daevadragon** [reply]
Hey again, sorry but even with the update, this still isn't working as expected for me. The estimations aren't disappearing anymore, but I'm not getting the effect to override to a different table. Anything I need to setup differently here? Same result on tokens that are using linked or unlinked actor data
![image](https://github.com/mclemente/healthEstimate/assets/114616897/e4043736-12b0-473a-bc5f-6779d63d240f)

**Reply by mclemente** [reply]
Try `effects.name === "Stable"` instead.

That is probably because the Effect has no actual Statuses set (e.g. the "Dead" effect has a Status, but a simple Effect doesn't). The sample code specifically mentions it is an effect that "set them unconscious", I'll make it more clear it means it adds an status effect and add an example that simply targets the name of an effect.

**Reply by daevadragon** [reply]
Here's what I'm seeing with that as the JS rule
![image](https://github.com/mclemente/healthEstimate/assets/114616897/5b7d5d65-c39a-4917-b1db-1077aaa16401)

And here's what I get trying to use the sample code
![image](https://github.com/mclemente/healthEstimate/assets/114616897/8b46454c-c6a8-45ad-ba15-41590404850c)

**Reply by mclemente** [reply]
Oof, I meant to say "replace `effect.status.has("unconscious")` for `effect.name === "Stable"`", not the whole JS Rule, sorry.

`actor.effects.find((effect) => effect.name === "Stable")`

**Reply by daevadragon** [reply]
Got it, thank you. I don't know if this is system dependent, but my game wanted "label" instead of "name" as well. 

Now I have to ask, the ignore color box seems to imply that there's a way to setup color on the label based on the estimation table being used. Is that the intended functionality, and how does one access that if so?

`actor.effects.find((effect) => effect.label === "Stable")`
![image](https://github.com/mclemente/healthEstimate/assets/114616897/d7c446e9-8690-4b97-ad9c-540e5806de14)
